components/scream/src/share/io/scream_io_utils.hpp

```diff
@@ -44,11 +45,58 @@ inline OutputAvgType str2avg (const std::string& s) {
 struct IOControl {
   // A non-positive frequency can be used to signal IO disabled
   int frequency = -1;
-  int nsteps_since_last_write;
+  int nsteps_since_last_write;  // Needed when updating output data, such as with the OAT::Average flag
+  util::TimeStamp& timestamp_since_last_write;
```

Comment:
I wouldn't call it "since" last write. This is the timestamp _of_ the last write. If I read "since", I think that this is a timestamp difference (which is dangerous to use, since depending on when t0 is, t1-t0 may consist of a different number of months/years).